src/components/CheckboxWithTooltip/CheckboxWithTooltipForMobileWebAndNative.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {View} from 'react-native';
+import Checkbox from '../Checkbox';
+import {propTypes, defaultProps} from './CheckboxWithTooltipPropTypes';
+import Growl from '../../libs/Growl';
+import CONST from '../../CONST';
+import compose from '../../libs/compose';
+import withWindowDimensions from '../withWindowDimensions';
+
+class CheckboxWithTooltipForMobileWebAndNative extends React.Component {
+    constructor(props) {
+        super(props);
+        this.onPress = this.onPress.bind(this);
+    }
+
+    componentDidUpdate() {
+        if (this.props.toggleTooltip) {
+            Growl.show(this.props.text, CONST.GROWL.WARNING, 3000);
+        }
+    }
+
+    /**
+     * Show warning modal on mobile devices since tooltips are not supported when checkbox is disabled.
+     */
+    onPress() {
```

Comment:
This method should be named for what it does (not what it handles). The only name I could come up with is `showGrowlAndTriggerOnPress`